src/Routes/LifecyclePage/LifecyclePage.tsx

```diff
@@ -14,14 +14,22 @@ const LifecyclePage = () => {
 
   return (
     <React.Fragment>
-      <PageHeader>
+      <PageHeader className="pf-u-pb-0">
         <PageHeaderTitle title="Life Cycle" />
       </PageHeader>
       <Alert
         id="lifecycle-warning"
         variant="warning"
         title="Dates are approximations and subject to change."
         component="h2"
+        className="no-shadow-alert"
+        style={{
+          boxShadow: 'none',
+          borderTop: 'none',
+          borderLeft: 'none',
+          paddingLeft: '28px',
+          marginTop: '-16px',
+        }}
```

Comment:
```suggestion
        isInline
        isPlain
        style={{ paddingLeft: '28px' }}
        }}
```
I remembered I once saw this[ `plain` and `inline` variant.](https://v5-archive.patternfly.org/components/alert#plain-inline-alert-variants):

![Screenshot From 2025-05-02 13-53-10](https://github.com/user-attachments/assets/36e40008-e5da-418d-8b53-52c85d6b3012)

Reply:
Works like a charm!